app/src/main/java/com/depromeet/team6/data/background/AlarmScheduler.kt

```
@@ -83,6 +85,23 @@ object AlarmScheduler {
+        val alarmManager = context.getSystemService(Context.ALARM_SERVICE) as AlarmManager
+        // 잠금화면 포그라운드 서비스 해제
+        val lockAlarmIntent = Intent(context, LockService::class.java)
+        val lockAlarmPendingIntent = PendingIntent.getForegroundService(
+            context,
+            ALARM_START_ID,
+            lockAlarmIntent,
+            PendingIntent.FLAG_UPDATE_CURRENT or PendingIntent.FLAG_IMMUTABLE
+        )
+        try {
+            alarmManager.cancel(lockAlarmPendingIntent)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)